examples/eplb/eplb_deepseek.py

```diff
@@ -0,0 +1,205 @@
+# SPDX-License-Identifier: Apache-2.0
+"""
+Expert parallelism load balancer (EPLB) for vLLM.
+The rearrangement algorithm is adapted from
+[DeepSeek EPLB](https://github.com/deepseek-ai/eplb).
+"""
+from typing import Tuple
+
+import torch
+
+
```

Comment:
is there any way to just import `https://github.com/deepseek-ai/eplb` instead of copy the code?